### PR TITLE
Fixes some EZ issues

### LINKED ISSUE #14026: Warning on statue.dm
`. == ..()` should be `. = ..()`
@AnturK 
https://github.com/tgstation/-tg-station/blob/master/code/modules/mob/living/simple_animal/hostile/statue.dm#L225

### LINKED ISSUE #14027: Santabag suicide_act has the wrong path
`/obj/item/weapon/storage/santabag/suicide_act(mob/user)` should be `/obj/item/weapon/storage/backpack/santabag/suicide_act(mob/user)`
@bgobandit

### PR DESCRIPTION
**Pull request by PKPenguin321**
Fixes the path for santabag/suicide_act, it was an ez fix so here you go
Fixes an extra = in statues.dm, also an ez fix. @tkdrg learn to use github inbuilt editor

Fixes https://github.com/tgstation/-tg-station/issues/14027
Fixes https://github.com/tgstation/-tg-station/issues/14026

### CODE REVIEW

**Comment by tkdrg** [comment]
but i'm 2lazy

File: code/modules/mob/living/simple_animal/hostile/statue.dm [hunk 1]
```diff
@@ -222,6 +222,6 @@
 	faction -= "neutral"
 
 /mob/living/simple_animal/hostile/statue/restrained()
-	. == ..()
```
Comment (duncathan) [inline code/modules/mob/living/simple_animal/hostile/statue.dm]: Lol

Comment (PKPenguin321) [inline code/modules/mob/living/simple_animal/hostile/statue.dm]: i dont get what is funny
did i do it wrong :sob:

Comment (duncathan) [inline code/modules/mob/living/simple_animal/hostile/statue.dm]: No you did it properly don't worry
It's just that the mistake you fixed was funny

**Pull request merged**